scrapy_zyte_api/providers.py

```diff
@@ -183,6 +194,11 @@ async def __call__(  # noqa: C901
             )
             results.append(browser_response)
 
+        if screenshot_requested:
+            screenshot_b64 = api_response.raw_api_response["screenshot"]
+            screenshot = Screenshot(body=b64decode(screenshot_b64.encode()))
```

Comment:
What do you think about moving this code to Screenshot itself? E.g. from_base64 classmethod, or maybe even some converter for the field (though probably not)?